### PR TITLE
New "Try it Out" experience

### PR DESCRIPTION
**Pull request by ismailmustafa**
This implements the "Try it Out" experience from this [Figma](https://www.figma.com/file/WLjLwZGP0jN1RNXSPPnB7P/Embedded-Interactive-Demo?node-id=0%3A1&t=zZrygRQ7e4JIhrHb-1). Please "Try it Out" and give feedback.

Click [here](https://developer-docs-git-devx-1148-beyondidentity.vercel.app/try-it-out) to see the experience.

### CODE REVIEW

**Comment by turnipdabeets** [comment]
The UI is blocked when displaying the username and scrolling through passkey details:

Screenshot 2023-01-11 at 1 20 43 PM

File: src/pages/try-it-out/index.jsx [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+        behavior: 'smooth'
+      });
+    });
+  };
+
+  return (
+    <div className={parentClassNames}>
+      <div className={classNames(padding["mt-1"], padding["mb-1"])}>
+        <h1>Try it Out</h1>
+        <p>Developers can use the features below to explore what's possible using the Beyond Identity platform before having to write any code.</p>
```
Comment (turnipdabeets) [inline src/pages/try-it-out/index.jsx:82]:
```suggestion
        <p>Explore what's possible using the Beyond Identity platform before having to write any code.</p>
```

File: src/pages/try-it-out/index.jsx [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+        <p>The three steps below allow you to:</p>
+        <ol>
+          <li><strong>Register a New User:</strong> We help developers manage user accounts and associate passkeys with each.</li>
```
Comment (turnipdabeets) [inline src/pages/try-it-out/index.jsx:85]:
```suggestion
          <li><strong>Register a new user:</strong> We help developers manage user accounts and associate passkeys with each.</li>
```

**Comment by turnipdabeets** [comment]
This is really cool!

**Comment by ismailmustafa** [comment]
> The UI is blocked when displaying the username and scrolling through passkey details:
> 
> Screenshot 2023-01-11 at 1 20 43 PM

@turnipdabeets I made a few updates to this to improve it.

**Comment by Will5** [comment]
LGTM. The only issue I noticed is that it can hang and not move forward. In this case, I saw a notification in Slack and went to read it. This is the error I saw in the console:

> Uncaught (in promise) Error: HandleUrl(BindCredential(AuthLib(AssertionEncode(Hal(Internal(AnyhowWrapper(NotAllowedError: The operation is not allowed at this time because the page does not have focus.)))))))

I ultimately refreshed and everything was fine. Probably an edge case, but maybe there is something we can do with exceptions?

**Comment by ismailmustafa** [comment]
> LGTM. The only issue I noticed is that it can hang and not move forward. In this case, I saw a notification in Slack and went to read it. This is the error I saw in the console:
> 
> > Uncaught (in promise) Error: HandleUrl(BindCredential(AuthLib(AssertionEncode(Hal(Internal(AnyhowWrapper(NotAllowedError: The operation is not allowed at this time because the page does not have focus.)))))))
> 
> I ultimately refreshed and everything was fine. Probably an edge case, but maybe there is something we can do with exceptions?

Good catch! I added a try/catch around all SDK function calls: https://github.com/gobeyondidentity/developer-docs/pull/203/commits/a25ace29bbf196928922ed3a36f6b2a5d35c9fdf

**Review by nmelo — APPROVED** [review]
LGTM

**Review by Will5 — APPROVED** [review]

**Pull request merged**